### ISSUE TITLE
Inconsistent whitespace in multi-line string literal

### ISSUE THREAD

**Issue by QF5690** [nicklockwood/SwiftFormat#208]
I think there is some problem with parsing multiline strings.
Valid code:
```
func test() -> String {
  return """
  \(String(describing: 4))
  """
}
```
Error:
```
warning: inconsistent whitespace in multi-line string literal at 3:22 in main.swift
error: failed to format any files
```
Version:
```
$ swiftformat --version
swiftformat, version 0.32.1
```

**Comment by nicklockwood** [comment]
@QF5690 thanks - I've found the issue. Seems to be caused by the space between "describing:" and "4". The parser is wrongly interpreting that as indent space. Should be easy to fix.

**Comment by nicklockwood** [comment]
@QF5690 fixed in 0.32.2
